src/index.html

```diff
@@ -13,6 +13,42 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__info">
+        <img
+          class="product-card__image"
+          src="images/imac.jpeg"
+          alt="product"
+        />
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="product-card__code">Product code: 195434</p>
+      </div>
+      <div class="product-card__reviews">
+        <div class="product-card__reviews-stars">
+          <div class="product-card__star product-card__star--1--filled"></div>
+          <div class="product-card__star product-card__star--2--filled"></div>
+          <div class="product-card__star product-card__star--3--filled"></div>
+          <div class="product-card__star product-card__star--4--filled"></div>
+          <div class="product-card__star product-card__star--5"></div>
+        </div>
+        <div class="product-card__reviews-total">Reviews: 5</div>
+      </div>
+      <div class="product-card__price">
+        <p class="product-card__price_title">Price:</p>
```

Comment:
Class naming: According to BEM methodology, the element's class name should be 'product-card__price-title' instead of 'product-card__price_title'. Only use single underscores to separate blocks from elements and double dashes for modifiers.